habushu-maven-plugin/src/test/resources/specifications/dependency-management.feature

```diff
@@ -53,7 +53,7 @@ Feature: Test dependency management capabilities to help align package versions
       | packageFooTest | ^1.2.7             |
       | packageBarTest | ^0.9.0             |
 
-  Scenario: Inactive managed dependencies are skipped
+  Scenario Outline: Inactive managed dependencies are skipped
```

Comment:
Simple syntax fixes. Cucumber seems smart enough to interpret Scenarios as Outlines based on whether Examples are given either way.